CMakeLists.txt

```diff
@@ -432,6 +432,48 @@ target_link_libraries(EventProcessorTests ${Boost_LIBRARIES}
 
 add_test(EventProcessor ${CMAKE_BINARY_DIR}/EventProcessorTests --log_sink=EventProcessorTests.log --report_sink=EventProcessorTests.report)
 
+add_executable(ProcessTreeTests
+        auoms_version.h
+        ProcessTreeTests.cpp
+        Event.cpp
+        TextEventWriter.cpp
+        RawEventProcessor.cpp
+        RawEventAccumulator.cpp
+        RawEventRecord.cpp
+        Signals.cpp
+        Logger.cpp
+        Config.cpp
+        UserDB.cpp
+        RunBase.cpp
+        ProcessInfo.cpp
+        ProcFilter.cpp
+        ProcessTree.cpp
+        FiltersEngine.cpp
+        StringUtils.cpp
+        TempDir.cpp
+        TestEventData.cpp
+        TranslateRecordType.cpp
+        TranslateSyscall.cpp
+        TranslateFieldType.cpp
+        TranslateField.cpp
+        TranslateArch.cpp
+        Interpret.cpp
+        ExecveConverter.cpp
+        Metrics.cpp
+        Cache.h
+        CmdlineRedactor.cpp
+        FileUtils.cpp
+)
```

Comment:
The following can be removed from this file list:
        TextEventWriter.cpp
        RawEventProcessor.cpp
        RawEventAccumulator.cpp
        RawEventRecord.cpp
        TempDir.cpp
        TestEventData.cpp
        TranslateRecordType.cpp
        TranslateSyscall.cpp
        TranslateFieldType.cpp
        TranslateField.cpp
        TranslateArch.cpp
        Interpret.cpp
        Metrics.cpp
        Cache.h
        CmdlineRedactor.cpp
        FileUtils.cpp